pages/api/workouts/[id].js

```diff
@@ -13,11 +13,13 @@ export default async function handler(req, res) {
       res.status(200).json({
         id: workout._id,
         name: workout.name,
+        exercises: [{ exerciseName: workout.name }],
```

Comment:
```suggestion
        exercises: [
        { exerciseName: workout.exercises[0]?.name }, 
        { exerciseName: workout.exercises[1]?.name }, 
        { exerciseName: workout.exercises[2]?.name }, 
        { exerciseName: workout.exercises[3]?.name }, 
        { exerciseName: workout.exercises[4]?.name }, 
        { exerciseName: workout.exercises[5]?.name }, 
        { exerciseName: workout.exercises[6]?.name }, 
        { exerciseName: workout.exercises[7]?.name }, 
        { exerciseName: workout.exercises[8]?.name }, 
        { exerciseName: workout.exercises[9]?.name }, 
        { exerciseName: workout.exercises[10]?.name },
        ],
```
Vorsicht, sonst bekommst du hier wieder "pushy workout" 
siehe [hier](https://github.com/ufelgen/capstone-project/blob/main/pages/api/words/index.js)